doc/nrf/gs_modifying.rst

```diff
@@ -213,14 +223,19 @@ Changes are picked up immediately, and you do not need to re-open the project in
 While it is possible to edit the :file:`.config` file directly, you should use SES or a tool like menuconfig or guiconfig to update it.
 These tools present all available options and allow you to select the ones that you need.
 
+To edit the file in |VSC|, use the nRF Kconfig extension.
+This extension organizes the Kconfig option in a hierarchical list and lets you select and deselect the desired options.
+To save the changes made using the nRF Kconfig extension, click the :guilabel:`Save` button.
+Read the `nRF Kconfig`_ documentation for more information.
+
 To edit the file in SES, select :guilabel:`Project` > :guilabel:`Configure nRF Connect SDK Project`.
 If your application contains more than one image (see :ref:`ug_multi_image`), you must select the correct target.
 To configure the parent image (the main application), select :guilabel:`menuconfig`.
 The other options allow you to configure the child images.
 
 See :ref:`zephyr:menuconfig` in the Zephyr documentation for instructions on how to run menuconfig or guiconfig.
 
-To locate a specific configuration option, use the filter (:guilabel:`Jump to` in menuconfig and guiconfig).
+To locate a specific configuration option, use the filter (:guilabel:`Jump To Item` in nRF Kconfig or :guilabel:`Jump to` in menuconfig and guiconfig).
```

Comment:
The "Jump to Item" options is only visible on options you have already enabled and which are showing up in the Changes tab. If you want to search for a new kconfig option to enable, it's better to type in the name of the option in the search field.

Reply:
Edited. Please check.